src/components/Checkbox/index.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function Smiley() {
+  return (
+    <div>
+      <input type="checkbox" />
```

Comment:
the component itself is ok, but please bear in mind that the way it is implemented, developers will not be able to get the value of the checkbox, as you didn't provide an onChange listener prop that the user can listen on. With onChange, you will probably also need to pass in a `checked` prop which gets updated on change